frontend/src/styles.scss

```diff
@@ -46,6 +46,9 @@ body {
   .business-registration__select {
     background-color: #666060;
   }
+  .search-toggle-element {
```

Comment:
is this style setting necessary here? It is conflicting with the component.scss.
It is encouraged to use styling in the components' scss

Reply:
This styling element is dark-mode controlled. Again.. opportunities.